OpenRA.Game/Platform.cs

```diff
@@ -108,6 +108,13 @@ public static string ResolvePath(string path)
 			if (path.StartsWith("./") || path.StartsWith(".\\"))
 				path = GameDir + path.Substring(2);
 
+			if (path.Contains("@"))
+			{
+				var mod = path.Substring(0, path.IndexOf('@'));
+				var file = path.Contains("@ ") ? path.Substring(path.IndexOf("@ ") + 2) : string.Empty;
+				path = Path.Combine(ModMetadata.CandidateModPaths[mod], file);
+			}
```

Comment:
This would be:
```c#
			if (path.StartsWith("$"))
			{
				var mod = path.Substring(1, path.IndexOf('/'));
				var file = path.Contains("/") ? path.Substring(path.IndexOf("/") + 1) : string.Empty;
				path = Path.Combine(ModMetadata.CandidateModPaths[mod], file);
			}
```